db/migrate/20161220020802_add_column_to_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :resize_thumb, :boolean, default: false, null: false
```

Comment:
Ah, this column will be not used I think because carrierwave doesn't store path for versions